### PR TITLE
Updated to Xamarin.Forms 5.0. and added simple Vibration API.

### PR DESCRIPTION
**Pull request by ErlingMK**
## Todo List

- [x] I have tested on an Android device.
- [x] I have tested on an iOS device.
- [ ] I have added unit tests

>*Please add pictures / Gifs if possible*
>*Todo List can be checked by putting a `X` inside the brackets*
>*Remember to update our `wiki` after this PR is merged*

### CODE REVIEW

**Review by haavamoa — CHANGES_REQUESTED** [review]
Update nuspec min dependencies

File: src/DIPS.Xamarin.UI/Controls/Slidable/SlidableLayout.cs [hunk 1]
```diff
@@ -347,7 +355,7 @@ public bool WidthIsProportional
         /// <summary>
         /// Set this to true if you want a small vibration every time the index changes.
         /// </summary>
-        public bool VibrateOnSelectionChanged { get; set; }
+        public bool VibrateOnSelectionChanged => true;
```
Comment (haavamoa) [inline src/DIPS.Xamarin.UI/Controls/Slidable/SlidableLayout.cs:358]: Are consumer able to turn it off now?

Comment (ErlingMK) [reply src/DIPS.Xamarin.UI/Controls/Slidable/SlidableLayout.cs:358]: Nope, that was not supposed to be in the PR. Will update

File: src/DIPS.Xamarin.UI.Android/DIPS.Xamarin.UI.Android.csproj [hunk 2]
```diff
@@ -55,17 +57,20 @@
     <Compile Include="Resources\Resource.designer.cs" />
     <Compile Include="Properties\AssemblyInfo.cs" />
     <Compile Include="InternalButtonRenderer.cs" />
+    <Compile Include="VibrationService.cs" />
   </ItemGroup>
   <ItemGroup>
     <None Include="Resources\AboutResources.txt" />
   </ItemGroup>
   <ItemGroup>
+    <PackageReference Include="Xamarin.AndroidX.MediaRouter" Version="1.2.0" />
```
Comment (ErlingMK) [inline src/DIPS.Xamarin.UI.Android/DIPS.Xamarin.UI.Android.csproj:66]: Do these have to be included in the nuspec? @haavamoa

Comment (haavamoa) [reply src/DIPS.Xamarin.UI.Android/DIPS.Xamarin.UI.Android.csproj:66]: If its not a transitive dependency, then yes.

**Comment by ErlingMK** [comment]
Need to add some API checks to some of the Android methods. If not, min version of Android would be API level 29.

**Review by haavamoa — APPROVED** [review]

**Pull request merged**